content/en/docs/concepts/services-networking/endpoint-slices.md

```diff
@@ -93,6 +93,46 @@ EndpointSlices support three address types:
 * IPv6
 * FQDN (Fully Qualified Domain Name)
 
+### Conditions
+
+The EndpointSlice API stores conditions about endpoints that may be useful for consumers.
+The three conditions are `ready`, `serving`, and `terminating`.
+
+#### Ready
+
+`ready` is a condition that maps to a Pod's `Ready` condition. A running Pod with the `Ready`
+condition set to `True` should have this EndpointSlice condition also set to `true`. For
```

Comment:
@andrewsykim Are the capitalizations of `Ready` and `True` here correct? Based on line 108 ("`ready` condition"), it's not immediately clear to me.

Reply:
yeah it's a little bit confusing because we are talking about the conditions of two separate APIs. `ready` condition in EndpointSlice is lower-case because it is a json/yaml field. `Ready` condition for Pods is upper-case because it is a codified value for a pod condition.